src/reactive/runner_manager.py

```diff
@@ -41,45 +49,52 @@ def reconcile(quantity: int, mq_uri: str, queue_name: str) -> int:
     Returns:
         The number of reactive runner processes spawned.
     """
-    current_quantity = _get_current_quantity()
+    pids = _get_pids()
+    current_quantity = len(pids)
     logger.info("Current quantity of reactive runner processes: %s", current_quantity)
     delta = quantity - current_quantity
     if delta > 0:
-        logger.info("Will spawn %d new reactive runner processes", delta)
+        logger.info("Will spawn %d new reactive runner process(es)", delta)
         _setup_logging_for_processes()
         for _ in range(delta):
             _spawn_runner(mq_uri=mq_uri, queue_name=queue_name)
     elif delta < 0:
-        logger.info(
-            "%d reactive runner processes are running. "
-            "Will skip spawning. Additional processes should terminate after %s.",
-            current_quantity,
-            REACTIVE_RUNNER_TIMEOUT_INTERVAL,
-        )
+        logger.info("Will kill %d process(es).", -delta)
```

Comment:
What causes this difference? Are runners running too long?

Reply:
Yes, it is not implemented yet, but the upcoming PR will spawn runners in the subprocess and wait until a job has been picked up. 
The user can set the number of runner processes per config option - so it could happen that there are more processes running in the background than expected and we have to downsclae.
Initially a timeout approach was suggested, but after internal discussions with @jdkandersson we decided to kill superfluous processes.

Reply:
I am resolving this conversation, feel free to reopen.